packer/builder/azure/arm/config_test.go

```diff
@@ -53,23 +53,30 @@ func TestConfigShouldDefaultVMSizeToStandardA1(t *testing.T) {
 
 func TestUserShouldProvideRequiredValues(t *testing.T) {
 	builderValues := make(map[string]string)
-	var lastElement = requiredConfigValues[len(requiredConfigValues)-1]
 
+	// Populate the dictionary with all of the required values.
 	for _, v := range requiredConfigValues {
 		builderValues[v] = "--ignored--"
```

Comment:
maybe something else than 'ignored', because this suggests to some readers that this key is ignored, whereas only the content is ignored, Maybe `"--some-value--"` ?